tests/runtime/macro

```diff
@@ -77,3 +77,7 @@ EXPECT (2, 6)
 NAME it can be part of an expression passed to a macro
 PROG macro add_one($x) { $x + 1 } begin { $a = add_one(add_one(1) + 1); print($a);  }
 EXPECT 4
+
+NAME it expands idents if there is a matching macro
+PROG macro one() { $x = 1; $x } begin { $a = one; print($a);  }
```

Comment:
I guess this is one "gotcha". If you *know* that `one()` is a "function", then you might be trying to assign a function pointer? I think this is just about clear documentation and explanation of macros.

Reply:
Well at the moment we don't have named functions that you can get the pointer to. In the future if we do have functions (or rather expose them to users) then the addition of a `&` or `addr()` could solve this. This also pertains to your other comment about addressing potential conflicts between macros and functions, which I'll fix.

Reply:
Yeah, I think this is just a gotcha to keep in mind. We should probably split the documentation, and have a good explanation of functions and macro expansion. I don't think it belongs in a man page.